AUTHORS

```diff
@@ -0,0 +1,8 @@
+# This is the list of Mail and Packages's significant contributors.
+#
+# This does not necessarily list everyone who has contributed code,
+# especially since many employees of one corporation may be contributing.
+# To see the full list of contributors, see the revision history in
+# source control.
+@moralmunky
```

Comment:
Just need your info here @moralmunky

Reply:
Done, just waiting for the checks to complete.